cmd/root.go

```diff
@@ -32,18 +33,40 @@ var RootCmd = &cobra.Command{
 	Short: "A FTP server that persists all data to Beyond Storage.",
 	Long:  "A FTP server that persists all data to Beyond Storage.",
 	Run: func(cmd *cobra.Command, args []string) {
-		if versionFlag {
-			fmt.Fprintf(os.Stdout, "BeyondFTP version %s\n", constants.Version)
-			return
+		if cfgDebugFlag {
+			pprof.StartPP()
 		}
-
 		c := config.LoadConfigFromFilepath(cfgFileFlag)
 		s, err := server.NewFTPServer(c)
 		check.ErrorForExit("server init error", err)
 		StartServer(s)
 	},
 }
 
+var versionCmd = &cobra.Command{
```

Comment:
We do not need a version command here. Since `cobra` support a `Version` field in `Command`, you can just init the `constants.Version` to it. See [this](https://github.com/spf13/cobra/blob/master/command.go#L96) as a reference.

Reply:
Like:
```go
var RootCmd = &cobra.Command{
	// other fields
	Version: constants.Version,
}

Reply:
In this way, we can also remove `version` flag.

Reply:
sure, thanks